### PR TITLE
Resprite serbian overcoat, change serb coat category, add serb coat to loadout menu, tweak serb coat values

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request
image
(no image with better buttons)
4 Points in loadout menu

	melee = 20,
	bullet = 25,
	energy = 10,
	bomb = 5,
	bio = 0,
	rad = 0

## Why It's Good For The Game

serb coat is rare as FUCK(found it only once despite playing for half a year) and it's not good enough for such rarity, some people don't even know it exists
so let's make it far more common, but also change the armor values.
also looks better
## Changelog
:cl:
add: serb coat in loadout menu
tweak: serb coat armor values lowered
imageadd: serb coat repsrite(by Armolitsky)
/:cl:

### CODE REVIEW

**Comment by Kegdo** [comment]
GOD DAMN YOU SHITYARD

**Comment by L74814** [comment]
just buy the pack from the vendor lmao
git gud

**Comment by Kegdo** [comment]
> just buy the pack from the vendor lmao git gud

hmmm yes, I will use 1800+ creds to get a shitty greatcoat

**Comment by L74814** [comment]
> > just buy the pack from the vendor lmao git gud
> hmmm yes, I will use 1800+ creds to get a shitty greatcoat

lack of money, job issue

**Comment by Kegdo** [comment]
> lack of money, job issue

lack of common sense there is also a better armor vest in the same kit
image

**Comment by zombiemimic** [comment]
what kind of crack you smoking?
>better armor vest

that doesn't have pockets or laser protection

**Comment by Kegdo** [comment]
I am going to commit multiple atrocities in Botswana

**Comment by Humonitarian** [comment]
Unrefactors my refractors, this is backwards Eris

**Comment by Kegdo** [comment]
maybe your refactors were simply bad :mitosis:

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -126,18 +126,6 @@
 /obj/item/clothing/suit/storage/greatcoat/ironhammer
 	icon_state = "greatcoat_ironhammer"
 
-/obj/item/clothing/suit/storage/greatcoat/serbian_overcoat
```
Comment (Humonitarian) [inline code/modules/clothing/suits/armor.dm:129]: Unrefactors my refactor that united all greatcoats and overcoats as the "Greatcoat" type, which encompasses armoured and protective jackets

Unless there's a good reason for it, this should be undone

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -332,6 +332,26 @@
 	)
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
 
+/obj/item/clothing/suit/storage/serbian_overcoat
```
Comment (Humonitarian) [inline code/modules/clothing/suits/miscellaneous.dm:335]: Lacks most flags jackets and especially coats possess, such as temperature resistance, spawn tags, coverage and price

File: code/modules/client/preference_setup/loadout/lists/suits.dm [hunk 3]
```diff
@@ -84,6 +84,20 @@
 	)
 	gear_tweaks += new /datum/gear_tweak/path(longcoat)
 
+/datum/gear/suit/serbian_overcoat
+	display_name = "serbian overcoat"
+	path = /obj/item/clothing/suit/storage/serbian_overcoat
+	cost = 4 // decent armor values
```
Comment (Humonitarian) [inline code/modules/client/preference_setup/loadout/lists/suits.dm:90]: Risks creating a new meta-pick for the loadout, a better solution would be tweaking the serb crates to be more affordable, no?

**Review by Humonitarian — COMMENTED** [review]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by Kegdo** [comment]
i have been copypasta'd, shit

**Comment by infraredbaron** [comment]
why the buttons are different on icon and state sprite?

**Comment by Kegdo** [comment]
because armolitsky didn't make an image with a new/better version, the buttons are fine in-game

**Comment by ValoTheValo** [comment]
You can buy the serb overcoat from the fucken vendor and a loadout 25 bullet item is not a good idea.

**Comment by Kegdo** [comment]
> You can buy the serb overcoat from the fucken vendor and a loadout 25 bullet item is not a good idea.

nobody buys it

4 points worth

**Comment by thedarkcrusader** [comment]
> > You can buy the serb overcoat from the fucken vendor and a loadout 25 bullet item is not a good idea.
> 
> nobody buys it
> 
> 4 points worth

A lot of people buy the crate but they dont use the coat they would pick the vest

**Comment by Kegdo** [comment]
fcuign
fine
I am currently too lazy to revert all the changes, so I will commit a pro gamer move instead

**Pull request closed without merging**